api.json

```diff
@@ -8934,7 +8934,7 @@
           "description": "The ID of the variation\u0027s associated item."
         },
         "ordinal": {
-          "type": "string",
+          "type": "integer",
```

Comment:
@tristansokol mentioned over Slack that this file mirrors one from an internal repository - and that he'd mirror those changes upstream.  So I suspect this PR might not be used at all.

That said, I'm curious why the `ordinal` field would be a `number` rather than `integer`. Ordinal values can not be floating points, so using `integer` seems appropriate.